com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -525,6 +525,13 @@ private void SpawnNetworkObjectLocallyCommon(NetworkObject networkObject, ulong
                 triggerInfo.TriggerData.Dispose();
                 m_Triggers.Remove(networkId);
             }
+
+            // propagate the IsSceneObject setting to child NetworkObjects
```

Comment:
is this a fix for scene objects only or does it also fix NetworkPrefabs spawned at runtime too?

Reply:
if so, how `IsSceneObject` prop fixes runtime NetworkPrefab spawning? I'm confused.

Reply:
`IsSceneObject` is an optional boolean. This simply propagates the parent setting (either case) to the children. So it makes sure it works for both situation. In practice, you are right, I believe only the spawned at runtime object were being affected.

So, maybe then comment should read "Propagating the IsSceneObject false setting to child NetworkObjects"